articlequality/extractors/ptwiki.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import re
+import traceback
+
+from .extractor import TemplateExtractor
+
+logger = logging.getLogger(__name__)
+
+POSSIBLE_LABELS = ("1", "2", "3", "4", "5", "6")
+
+
+def from_template(template):
+
+    if template.has_param('class'):
```

Comment:
We'll want check `if normalize_template_name == "marca de projeto"`.  We'll need to regex for parsing the template contents.  

Something like this will probably work: `\{\{marca de projeto\s*\|\s*[0-9]+(\s*\|\s*([a-z0-9-_ ]+)\s*\|\s*([0-9]+))+\}\}`

Reply:
Note that we'll be getting the project and class (0-5, *) from this one template, so we won't be processing the "label" from the "class" param below.

Reply:
So I can remove the try except block?